src/Serilog.Sinks.MSSqlServer/Configuration/Extensions/Hybrid/LoggerConfigurationMSSqlServerExtensions.cs

```diff
@@ -121,13 +123,15 @@ public static class LoggerConfigurationMSSqlServerExtensions
             IFormatProvider formatProvider = null,
             ColumnOptions columnOptions = null,
             IConfigurationSection columnOptionsSection = null,
-            ITextFormatter logEventFormatter = null) =>
+            ITextFormatter logEventFormatter = null,
+            LoggingLevelSwitch levelSwitch = null) =>
```

Comment:
Thanks @jonorossi for your review and thoughts.

Regarding binary compatibility. As you already mentioned, I don't think that this is a real goal for our sink. If users update libraries in their apps without recompiling, by just dropping in the new assemblies, they are in a deep red danger zone anyway. They will get problems with most libraries, so we don't need to bother about binary breaking changes.

I also don't like default value parameters. But I delibarately added levelSwitch to the extension method parameters rather than to the MSSQLSinkOptions class. The reason for that was, that levelSwitch is a common serilog parameter which most sinks have and most sinks (even if they have something like a sink options class) have levelSwitch as a parameter in their config extension methods. I wanted to keep things uniform throughout the serilog ecosystem. Another reason for plaing it in the method parameters is, that levelSwitch is connected to the parameter restrictedToMinimumLevel and therefore should be placed next to it.

That said, I took some time to consider your remarks. Finally I went with your suggestion and put LevelSwitch into the MSSqlServerSinkOptions class. We should not make an already bloated config method interface even more bloated.

Reply:
> If users update libraries in their apps without recompiling, by just dropping in the new assemblies, they are in a deep red danger zone anyway.

I more meant with NuGet using SemVer, if one package targets 6.1.1 and another 6.2.0, NuGet would upgrade you to 6.2.0, and one will break. I know some companies publish internal libraries as packages to an internal feed for other teams to consume, these libraries obviously get compiled separately.

> That said, I took some time to consider your remarks. Finally I went with your suggestion [...]

😃